metadata.tex

```diff
@@ -291,4 +386,6 @@
 
 \clearpage
 
+\include{sections/sentinels.tex}
```

Comment:
This is good for now, I'm just thinking out loud. Maybe we should create separate files with appropriate dirs, like `campaigns.tex`, `clash.tex`, `coop.tex`, etc.

Reply:
Scenarios could be listed in those files in a pre-defined order, while individual scenarios could reside in their own TeX files in appropriate dirs.

Reply:
Yes, I agree. We can adjust the structure as we include more missions